### PR TITLE
Run log filter threshold from xml

### LINKED ISSUE #108: Run log filter threshold from xml
Take over the filters from defined standard file appender from logback xml-config, if applicable. Else log with "INFO"-Level

### PR DESCRIPTION
**Pull request by ckittl**
Resolves #108

### CODE REVIEW

**Comment by ckittl** [comment]
Restart of rolling file appender does not work too great, yet. If required, I would have to look into this in detail.

@johanneshiry I assume this is the intended behavior? Or did you intend the log file in output directory only have "INFO"-Level and above? In that case I'd need to adapt the implementation.

**Review by sebastian-peter — DISMISSED** [review]
LGTM

**Comment by johanneshiry** [comment]
> Restart of rolling file appender does not work too great, yet. If required, I would have to look into this in detail.
> 
> 
> 
> @johanneshiry I assume this is the intended behavior? Or did you intend the log file in output directory only have "INFO"-Level and above? In that case I'd need to adapt the implementation.

The main intention is to **always** have a log file with the INFO-level in the output directory. If you want to have additional logs with more detailed log levels you can just adapt additional appenders in the xml-config. Therefore I would prefer not changing this behavior, but leave it to the user in the xml config to configure additional logger with e.g. debug level or below. 
Background: I always want at least **one** non-cluttered info log in the output dir. If you want debug log it can be **additionally** configured in the xml config file.

**Review by johanneshiry — CHANGES_REQUESTED** [review]
See comment regarding your question. If required we can discuss this further. Pls let me know if you want to do so.

**Comment by t-ober** [comment]
`
        rootLogger.iteratorForAppenders().asScala.foreach {
          case rf: RollingFileAppender[_] =>
            rf.getTriggeringPolicy.start()
            rf.start()
          case appender => appender.start()
        }
`
in LofbackConfiguration#default fixes the main issue.

File: CHANGELOG.md [hunk 1]
```diff
@@ -66,6 +66,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Renamed ChpData to ChpRelevantData [#494](https://github.com/ie3-institute/simona/issues/494)
 
 ### Fixed
+- Location of `vn_simona` test grid (was partially in Berlin and Dortmund)
```
Comment (danielfeismann) [inline CHANGELOG.md:69]: Does this apply to this PR?

**Review by danielfeismann — CHANGES_REQUESTED** [review]
Looks good to me, just one very minor question.

File: CHANGELOG.md [hunk 2]
```diff
@@ -66,6 +66,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
...
+- Take log file event filters from `logback.xml` when defining the run log appender
```
Comment (danielfeismann) [inline CHANGELOG.md:70]: Issue No.

**Review by danielfeismann — APPROVED** [review]
LGTM

**Pull request merged**